go.mod

```diff
@@ -135,10 +148,8 @@ require (
 )
 
 replace (
-	github.com/btcsuite/btcd => github.com/btcsuite/btcd v0.22.2 //indirect
-	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.13-ics
+	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.15-ics
```

Comment:
This bump is not SM breaking. Why combine it with a bump to ibc-go 4.3.0 which is ST breaking? Please create separate PRs.

Reply:
I've removed the change to ibc-go v4.3.0